src/xpk/core/capacity.py

```diff
@@ -99,6 +99,54 @@ def get_capacity_type(args) -> tuple[CapacityType, int]:
   return capacity_type, return_code
 
 
+def get_reservation_maintenance_interval(
+    reservation: str, zone: str, project: str
+) -> str:
+  """Get reservation maintenance interval.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  command = (
+      f'gcloud beta compute reservations describe {reservation}'
+      f' --project={project} --zone={zone} --format="value(specificReservation.instanceProperties.maintenanceInterval)"'
+  )
+  return_code, output = run_command_for_value(
+      command, 'Get reservation maintenance interval', None
+  )
+  if return_code != 0:
+    xpk_print(f'Get reservation maintenance interval ERROR {return_code}')
+    return ''
```

Comment:
This allows the process to continue after failing the command. Was this intended?

The same thing is in `get_reservation_placement_policy`.